preload/preload-resource-match.https.sub.html

```diff
@@ -0,0 +1,156 @@
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/common/utils.js"></script>
+<script src="/preload/resources/preload_helper.js"></script>
+<script>
+
+const crossOriginHost = 'https://{{host}}:{{ports[https][0]}}'
```

Comment:
We typically use [get-host-info.sub.js](https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/web_tests/external/wpt/common/get-host-info.sub.js;l=1?q=get-hos%20f:external%2Fwpt&ss=chromium) for those things, as it encapsulates the "sub" parts and is generally more readable.

Reply:
Yea, originally it needed to be as a sub.html, I'll revise